app/src/main/java/com/qudus/tudee/ui/composable/TudeeBadge.kt

```diff
@@ -0,0 +1,77 @@
+package com.qudus.tudee.ui.composable
+
+import android.util.Log
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.scaleIn
+import androidx.compose.animation.scaleOut
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.qudus.tudee.R
+import com.qudus.tudee.designSystem.theme.Theme
+
+@Composable
+fun TudeeTextBadge(
+    badgeNumber: String,
+    modifier: Modifier = Modifier,
+    contentColor: Color = Color(0x611F1F1F),
+    containerColor: Color = Color(0xFFF0F0F0)
+) {
+    Text(
+        modifier = modifier
+            .widthIn(min = 32.dp)
+            .background(containerColor, shape = RoundedCornerShape(percent = 50))
```

Comment:
I think u can use CircleShape here
```suggestion
            .background(containerColor, shape = CircleShape)
```

Reply:
> I think u can use CircleShape here

I assumed that without setting a specific size, using CircleShape would make it a full circle rather than just having fully rounded corners 🤦‍♂️. Thanks for pointing that out!